### PR TITLE
removes event-tags valid numeric

### PR DESCRIPTION
**Pull request by rashidsp**

### CODE REVIEW

File: tests/test_optimizely.py [hunk 1]
```diff
@@ -1847,23 +1802,14 @@ def test_track__invalid_attributes(self):
...
-    """ Test that expected log messages are logged during track when attributes are in invalid format. """
+    """ Test that expected log messages are logged during track when event_tag is in invalid format. """
 
     with mock.patch('optimizely.logger.SimpleLogger.log') as mock_logging:
       self.optimizely.track('test_event', 'test_user', event_tags='4200')
-
-    mock_logging.assert_called_once_with(enums.LogLevels.ERROR, 'Provided event tags are in an invalid format.')
+      self.optimizely.track('test_event', 'test_user', event_tags=4200)
+    mock_logging.assert_called_with(enums.LogLevels.ERROR, 'Provided event tags are in an invalid format.')
+    self.assertEqual(2, mock_logging.call_count)
```
Comment (oakbani) [inline tests/test_optimizely.py]: Does this ensure that each of the calls were made in separate track call?

File: tests/test_optimizely.py [hunk 2]
```diff
@@ -1022,57 +1022,6 @@ def test_track__with_event_tags__forced_bucketing(self):
     self._validate_event_object(mock_dispatch_event.call_args[0][0], 'https://logx.optimizely.com/v1/events',
                                 expected_params, 'POST', {'Content-Type': 'application/json'})
 
-  def test_track__with_deprecated_event_value(self):
```
Comment (oakbani) [inline tests/test_optimizely.py:1025]: Looks like your removal of this method decreased coverage for method isUserInExperiment. 
In other testcases we mock outer methods or isUserInExperiment so those lines do not get covered. Right a testcase to cover the missing lines.

**Review by oakbani — CHANGES_REQUESTED** [review]
Please fix code coverage and mocked assertion

**Comment by rashidsp** [comment]
Closing it; PR sent to Optimizely

**Pull request closed without merging**